Sandbox/src/Game/Interface/InventoryContainer.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <Square.h>
+#include <ctime>
+
+#include "Game/Interface/IInputContainer.h"
+#include "Game/Interface/Tooltip.h"
+#include "Game/Player/Inventory.h"
+
+class InventoryContainer : public IInputContainer
+{
+public:
+	const std::string CONTAINER_PATH = "Graphics/opened-menu.png";
```

Comment:
These ```const std::string``` seem like a bit of overkill but it's your preference lol

Reply:
I like keeping the file paths/constants in the .h file for easy configuration :)